h2o-r/tests/testdir_algos/targetencoder/runit_targetencoder_model.R

```diff
@@ -1,6 +1,10 @@
 setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
 source("../../../scripts/h2o-r-test-setup.R")
 
+doTestAndContinue <- function(testDesc, test) {
```

Comment:
Please use the already existing method `doTest`. This is copy-paste-modify. Not maintainable.

Reply:
@michalkurka sure, I'm aware of `doTest` method. Btw I wonder why we are ok with for example `get_partitioned_model_names` method in python then? But I agree that it is better to have utils that are being shared between all tests. The reason why I added this modification is that I don't wan't one-file-per-test approach. And I don't want to put all use cases into one test like we usually do in R. Maybe we should agree upon some util method and add it to test utils?

Reply:
I've found `doSuite` in utils so I think it will work for me.